debugger/src/main/scala/org.apache.daffodil.debugger.dap/Parse.scala

```diff
@@ -754,12 +771,14 @@ object Parse {
             Parse(
               args.schemaPath,
               in,
+              FileUtils.readFileToByteArray(args.dataPath.toFile),
```

Comment:
This will prevent you from being able to support files larger than 2GB. If this is important, you might want to just pass in the `File` and then the functions can call things like `Files.size()`, or `Files.read()` with offset to read just the parts of the file you want to output in diagnostics. That would avoid the 2GB limitation.